lib/cmps/base.ts

```diff
@@ -200,7 +200,7 @@ export class AutoConsentCMP extends AutoConsentCMPBase {
 
     async detectCmp() {
         if (this.rule.detectCmp) {
-            return this._runRulesParallel(this.rule.detectCmp);
+            return this._runRulesSequentially(this.rule.detectCmp);
```

Comment:
I see this was parallel even before #14, but somehow I always assumed this was sequential.. I guess for most rules it doesn't matter because they just match some selectors, but could you check that no rules actually rely on this? Like, evals or waits

Reply:
Looks like @noisysocks already did that check, so marking this as resolved.